src/defaultContent/styles/manage-keys.css

```
@@ -0,0 +1,204 @@
+    padding: 0.75rem 1.5rem;
+    text-decoration: none;
+    cursor: pointer;
+    text-align: center;
+    transition: all 0.2s ease-in-out, box-shadow 0.2s ease-in-out;
+    border-radius: 8px 8px 0 0;
+    width: 100%;
+    position: relative;
+    margin-bottom: -2px;
+    z-index: 0;
+    box-shadow: 0 2.5px 2.5px var(--bs-primary-border-subtle);
+    border: 1px solid var(--bs-primary-border-subtle);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)